tools/walletextension/rpcapi/blockchain_api.go

```diff
@@ -148,35 +159,47 @@ func (api *BlockChainAPI) GetStorageAt(ctx context.Context, address common.Addre
 			return nil, err
 		}
 		return userID, nil
+	case common.ListPrivateTransactionsCQMethod:
+		// sensitive CustomQuery methods use the convention of having "address" at the top level of the params json
+		userAddr, err := extractCustomQueryAddress(params)
+		if err != nil {
+			return nil, fmt.Errorf("unable to extract address from custom query params: %w", err)
+		}
+		resp, err := ExecAuthRPC[any](ctx, api.we, &ExecCfg{account: userAddr}, "eth_getStorageAt", address.Hex(), params, nil)
```

Comment:
let's add a todo to make this call a normal Ten RPC endpoint.
This "hack" should be purely a gateway convenience